### ISSUE TITLE
Doesn't activate on iOS 11.3.1 iPhone 8+

### ISSUE THREAD

**Issue by Anon2626** [jakeajames/OneHandWizardFix#2]
The app itself is blank but there's still the option within settings itself. When activating onehandwizard it showed the welcome popup or thanks for buying don't remember the exact message. After that message the action doesn't work nothing gets activated no message, no onehandwizard. I did install the fix for onehandwizard and activator. I also tried multiple different activator actions all with the same result.

**Comment by jakeajames** [comment]
Did you get any error while installing? And, did you use "OneHandWizard
Fix"? I replaced that with "OneHandWizard Installer+Fix", which patches
postinst instead.

Il giorno venerdì 4 gennaio 2019, Anon2626 <notifications@github.com> ha
scritto:

**Comment by Anon2626** [comment]
Yes just tried again with your updated fix. Here’s the error message I’m getting when installing OneHandWizard

https://imgur.com/a/Ckwqy9h

Want to add it also says package not correctly installed or something and Cydia asks if I want to ignore temporary or force remove.


Edit - also tried your twitter method 

ldid -e /bin/bash > ent.xml && ldid -Sent.xml /Library/dpkg/info/com.gx5.onehandwizard.postinst && /Library/dpkg/info/com.gx5.onehandwizard.postinst && rm ent.xml

Also for 
var/lib/dpkg/status change "install ok half-configured" to "install ok installed" for OneHandWizard I looked in Filza but can’t find it.

**Comment by jakeajames** [comment]
My updated package will install OneHandWizard automatically. Please remove
anything related to OneHandWizard and install the one that says
"OneHandWizard Installer+Fix", make sure to respring only when cydia shows
that option and maybe wait a few seconds before that too (due to how I'm
downloading OHW; i'm waiting for cydia to finish installing the fix and
only then i install ohw, you can't install something while something else
is installing sadly)

Il giorno giovedì 10 gennaio 2019, Anon2626 <notifications@github.com> ha
scritto:

**Comment by Anon2626** [comment]
Nothing is showing up now when trying to enable tweak and getting package half installed message in Cydia.


![Cydia install log](https://user-images.githubusercontent.com/46364972/50982572-7aa41100-14cb-11e9-935d-c257fe836537.png)

**Comment by Anon2626** [comment]
Nothing is showing up now when trying to enable tweak and getting package half installed message in Cydia.


Cydia log -> https://imgur.com/a/qXFxTDE

Blank App -> https://imgur.com/a/OI3Qg5B

Cydia error message -> https://imgur.com/a/TMTuC2i

**Comment by jakeajames** [comment]
Your jailbreak libraries are messed up, that screnshot is filled with
warnings & errors. I'd recommend a proper repo for these, like Elecubratus
on unc0ver.

Il giorno giovedì 10 gennaio 2019, Anon2626 <notifications@github.com> ha
scritto:
